### PR TITLE
Integrate a bunch of linters into travis and our environment.

### PR DESCRIPTION
**Pull request by geeknoid**
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/136)

### CODE REVIEW

**Comment by geeknoid** [comment]
This integrates stuff into Travis now. PTAL.

File: adapter/denyChecker/config_proto [hunk 1]
```diff
@@ -0,0 +1 @@
+bazel-genfiles/adapter/denyChecker
```
Comment (douglas-reid) [inline adapter/denyChecker/config_proto]: nit: do we want a comment explaining this file?

File: adapter/genericListChecker/BUILD [hunk 2]
```diff
@@ -29,10 +29,10 @@ load("@org_pubref_rules_protobuf//go:rules.bzl", "go_proto_library")
 
 go_proto_library(
     name = "genericListChecker",
+    output_to_workspace = False,
```
Comment (douglas-reid) [inline adapter/genericListChecker/BUILD]: nit: i think False is the default, so we can remove this line if we want.

**Comment by geeknoid** [comment]
Whops, that config_proto file should not be checked in, I'll remove it from
the PR.

I'll strip out all output_to_workspace in the build files.

**Pull request merged**